docs/src/capi/howto/intermediate-ca.md

```diff
@@ -0,0 +1,115 @@
+# How to use intermediate CAs with Vault
+
+By default, the ClusterAPI provider will generate self-signed CA certificates
+for the workload clusters.
+
+Follow this guide to prepare an intermediate Certificate Authority (CA) using
+HashiCorp Vault and then configure ClusterAPI to use the generated certificates.
+
+## Preparing Vault
+
+For the purpose of this guide, we are going to install HashiCorp Vault using
+snap and start a Vault server in development mode.
+
+```bash
```

Comment:
nit: Can you please remove the bash from the code snippets? Generally we leave them blank unless there is a real need so that the output looks the same across pages